requirements/requirements.txt

```
@@ -1,19 +1,19 @@
 colorama
-distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
+distrax
 flashbax~=0.1.0
 flax
 gigastep @ git+https://github.com/mlech26l/gigastep
 hydra-core==1.3.2
 id-marl-eval @ git+https://github.com/instadeepai/marl-eval
-jax
-jaxlib
+jax==0.4.25  # mabrax limiting the version here :/
```

I don't have a better comment but it's a suggestion 👀 
```suggestion
jax==0.4.25   # Fixed due to Mabrax dependency
```

---

I was being a bit cheeky :joy: